eisbuk-admin/src/pages/customer_area/CustomerNavigation/CustomerNavigation.tsx

```diff
@@ -67,7 +67,9 @@ const CustomerNavigation: React.FC = () => {
         __placeholder__,
         CustomerRoute.BookIce
       );
-      history.push(defaultRoute);
+      // here we're using `.replace` rather than `.push`
+      // to allow for back navigation (since it's an automatic redirect)
```

Comment:
I would say we use `.replace` because it's semantically correct. The weird behaviour was a consequence of misusage of the history api.

Here we don't want to add something to the history. We want to effectively change the current history entry.

No need to change the code, but I believe it's a distinction worth mentioning.